packages/mobile/src/components/Button/Button.tsx

```diff
@@ -22,10 +22,11 @@ export const CustomButton = ({
   variant,
   title,
   icon,
+  // testIcon,
   ...buttonProps
 }: CustomButtonProps) => {
   const isPrimary = variant === 'primary' ? true : false;
-  const iconSource = icon && ICONS[icon];
+  // const iconSource = icon && ICONS[icon];
```

Comment:
Do you expect to still need these in future?

Reply:
I didn't remove some comment that can helps make changes according to my inner todo list.